src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -27,6 +27,8 @@ import {startApm} from 'app/utils/apm';
 // SDK INIT  --------------------------------------------------------
 Sentry.init({
   ...window.__SENTRY__OPTIONS,
+  // TODO: Default `*` in ALLOWED_HOSTS doesn't work as we don't use globs in the SDK
+  whitelistUrls: 'http',
```

Comment:
It will break `settings.ALLOWED_HOSTS`, I used it for local testing only 😅

Reply:
Oops, thought I removed that